package.json

```diff
@@ -2,22 +2,25 @@
   "name": "chromium-bidi",
   "version": "0.4.2",
   "description": "An implementation of the WebDriver BiDi protocol for Chromium implemented as a JavaScript layer translating between BiDi and CDP, running inside a Chrome tab.",
-  "main": "index.ts",
   "scripts": {
     "build": "tsc -b src/tsconfig.json && npm run rollup",
     "clean": "rimraf lib",
     "e2e-headful": "npm run server-no-build -- --headless=false & npm run e2e-only",
     "e2e-headless": "npm run server-no-build & npm run e2e-only",
     "e2e-only": "python3 -m pytest --rootdir=tests",
     "e2e": "npm run e2e-headless",
-    "format": "npm run prettier & npm run yapf",
-    "lint": "npx prettier --check .",
-    "prepare": "npm run clean",
-    "prettier": "npx prettier --write .",
+    "lint": "npm run lint:eslint",
+    "lint:prettier": "prettier --check .",
+    "lint:eslint": "eslint --ext js --ext ts .",
+    "prepare": "npm run clean && npm run build",
+    "prettier": "npm run format:prettier",
+    "format": "npm run format:eslint && npm run format:prettier && npm run yapf",
+    "format:eslint": "eslint --ext js --ext ts --fix .",
+    "format:prettier": "prettier --write .",
     "rollup": "rollup --config configs/rollup.config.js",
     "server-no-build": "node lib/cjs/bidiServer/index.js",
     "server": "npm run build && npm run server-no-build --",
-    "test": "npm run unit",
+    "test": "npm run build && npm run unit",
```

Comment:
No need in building here then: https://github.com/GoogleChromeLabs/chromium-bidi/blob/f8d953fa7313c562f88471865a4fd18dc81f1715/.github/workflows/unit.js.yml#L23

Reply:
It requires building without ts-node to keep the same behaviour